modules/sql-engine/src/integrationTest/sql/identifiers/test_long_identifiers.test

```diff
@@ -0,0 +1,209 @@
+# name: sql/identifiers/test_long_identifiers.test
+# description: SQL feature F391 (Long identifiers)
+# group: [identifiers]
+
+# TODO: IGNITE-19703 Add cases for long identifiers for schema names.
+
+statement ok
+PRAGMA enable_verification
+
+# Create table with short identifiers for test simplicity purpose
+statement ok
+CREATE TABLE t (id INTEGER, val INTEGER, PRIMARY KEY (id))
+
+# Create table with long identifiers
+statement ok
+CREATE TABLE tableName_veryLoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongIdentifierOf128Characters (keyColumnName_veryLoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongIdentifierOf128Characters INTEGER, valueColumnName_veryLoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongIdentifierOf128Characters INTEGER, PRIMARY KEY (keyColumnName_veryLoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongIdentifierOf128Characters));
+
+statement error
```

Comment:
Could you please include an error message check here ? I can be done by using the following syntax
> statement error: Blah blah blah
> SELECT WRONG